src/hsv.temper.md

```diff
@@ -0,0 +1,8 @@
+# HSV and HSL Color Representation
+
+## HSV
+
+    // export let rgbToHsv(rgb: Matrix): Matrix {
+    //   let max = rgb.max(axis = 1);
+    //   rgb
+    // }
```

Comment:
Something I plan to work on sometime but still haven't gotten to. But got tired of keeping this placeholder local only.